src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/payments/client/FeeClient.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.payments.client;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ApplicationType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.fee.FeeResponse;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.payments.config.FeeServiceConfiguration;
+
+import java.net.URI;
+
+import static org.springframework.web.util.UriComponentsBuilder.fromHttpUrl;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class FeeClient {
+
+    private final FeeServiceConfiguration serviceConfig;
+    private final RestTemplate restTemplate;
+
+    public FeeResponse getApplicationFee(ApplicationType application) {
+        URI uri = buildUri(application);
+        log.info("Inside getApplicationFee, FeeResponse API uri : {} ", uri);
+        ResponseEntity<FeeResponse> response = restTemplate.getForEntity(uri, FeeResponse.class);
+        log.info("Fee response : {} ", response);
```

Comment:
check the respone - what it prints, anything sensitive? if yes we might need to print non sensitive data.